frontend/components/common/Label.jsx

```diff
@@ -14,10 +15,12 @@ Label.propTypes = {
   text: PropTypes.string.isRequired,
   color: PropTypes.string,
   bg: PropTypes.string,
+  icon: PropTypes.element,
 };
 Label.defaultProps = {
   bg: undefined,
```

Comment:
bg의 기본값이 defaultProps에 있어도 될 것 같습니다
```suggestion
  bg: 'red',
```